docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx

```diff
@@ -47,7 +47,7 @@ Returns `Promise<void>`.
 | --------- | -------- | ---------------------------- |
 | `tag`     | `string` | The cache tag to revalidate. |
 
-Returns `Promise<void>`. Learn more about [revalidating data](/docs/app/building-your-application/data-fetching/incremental-static-regeneration) or the [`revalidateTag()`](/docs/app/api-reference/functions/revalidateTag) function.
+`revalidateTag` does not return a value. Learn more about [revalidating data](/docs/app/building-your-application/data-fetching/incremental-static-regeneration) or the [`revalidateTag()`](/docs/app/api-reference/functions/revalidateTag) function.
```

Comment:
This does return a Promise per the type here https://github.com/vercel/next.js/blob/fc2519ea3d48393910caa6f49aaf90d6a70cacc1/packages/next/src/server/lib/incremental-cache/index.ts#L59-L61

Reply:
I believe the confusion is with the `revalidateTag` from `next/cache` which doesn't return a Promise

Reply:
Oh okey, my bad, thanks for clearing this up